bundles/org.openhab.binding.airparif/src/main/java/org/openhab/binding/airparif/internal/handler/AirParifBridgeHandler.java

```diff
@@ -0,0 +1,337 @@
+import org.openhab.binding.airparif.internal.api.AirParifDto.PollensResponse;
+import org.openhab.binding.airparif.internal.api.AirParifDto.Route;
+import org.openhab.binding.airparif.internal.api.AirParifDto.Version;
+import org.openhab.binding.airparif.internal.api.PollenAlertLevel;
+import org.openhab.binding.airparif.internal.api.Pollutant;
+import org.openhab.binding.airparif.internal.config.BridgeConfiguration;
+import org.openhab.binding.airparif.internal.deserialization.AirParifDeserializer;
+import org.openhab.binding.airparif.internal.discovery.AirParifDiscoveryService;
+import org.openhab.core.library.types.DateTimeType;
+import org.openhab.core.library.types.StringType;
+import org.openhab.core.thing.Bridge;
+import org.openhab.core.thing.ChannelGroupUID;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.Thing;
+import org.openhab.core.thing.ThingStatus;
+import org.openhab.core.thing.ThingStatusDetail;
+import org.openhab.core.thing.ThingUID;
+import org.openhab.core.thing.binding.BaseBridgeHandler;
+import org.openhab.core.thing.binding.ThingHandlerService;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.State;
+import org.openhab.core.types.UnDefType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * {@link AirParifBridgeHandler} is the handler for OpenUV API and connects it
+ * to the webservice.
+ *
+ * @author Gaël L'hopital - Initial contribution
+ *
+ */
+@NonNullByDefault
+public class AirParifBridgeHandler extends BaseBridgeHandler implements HandlerUtils {
+    private static final int REQUEST_TIMEOUT_MS = (int) TimeUnit.SECONDS.toMillis(30);
+    private static final Charset DEFAULT_CHARSET = StandardCharsets.UTF_8;
+    private static final String AQ_JOB = "Air Quality Bulletin";
+    private static final String POLLENS_JOB = "Pollens Update";
+    private static final String EPISODE_JOB = "Episode";
+
+    private final Logger logger = LoggerFactory.getLogger(AirParifBridgeHandler.class);
+    private final Map<String, ScheduledFuture<?>> jobs = new HashMap<>();
+    private final AirParifDeserializer deserializer;
+    private final HttpClient httpClient;
+
+    private BridgeConfiguration config = new BridgeConfiguration();
+    private @Nullable PollensResponse pollens;
+
+    public AirParifBridgeHandler(Bridge bridge, HttpClient httpClient, AirParifDeserializer deserializer) {
+        super(bridge);
+        this.deserializer = deserializer;
+        this.httpClient = httpClient;
+    }
+
+    @Override
+    public void initialize() {
+        logger.debug("Initializing AirParif bridge handler.");
+        config = getConfigAs(BridgeConfiguration.class);
+        if (config.apikey.isEmpty()) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                    "@text/offline.config-error-unknown-apikey");
+            return;
+        }
+        scheduler.execute(this::initiateConnexion);
```

Comment:
Please set Thing state to `Unknown` before starting the async initiate.

Reply:
Done